### DISCUSSION TITLE
Real time Database subscription, channel name

### DISCUSSION THREAD

**Discussion by JunzheFan** [supabase/supabase#31267] [Questions] [answered]
```
  const channelA = supabase
    .channel('schema-db-changes')
    .on(
      'postgres_changes',
      {
        event: '*',
        schema: 'public',
      },
      (payload) => console.log(payload)
    )
    .subscribe()
```
I am confused with the channel name parameters for listenting to database changes,  let's say if i have 10 users who installed my desktop application, and they are listening to the same table for changes, do I need to generate a different channel name myself for each of the user or they can share the same channel name? and in terms of unsubscribe, if I do `supabase.removeChannel(myChannel)` for a user user, will the other 9 users will also get unsubscribed if the channel name was the same when creating the client instance ?

**Comment by GaryAustin1** [comment, marked as answer]
For postgres changes the channel name is not really doing anything useful.   There is no interaction between the other clients and the name can be the same or different.
